core/sql/executor/ExStats.cpp

```diff
@@ -2741,13 +2749,21 @@ Int64 ExHbaseAccessStats::getNumVal(Int32 i) const
 
 NABoolean ExHbaseAccessStats::filterForSEstats(struct timespec currTimespec, Lng32 filter)
 {
-   blockTime_ = timer_.filterForSEstats(currTimespec);
-   if (blockTime_ >= filter)
-      return TRUE;
+   Int64 sumIOTime;
```

Comment:
The same logic seems duplicated between this class and ExHdfsScanStats.  Maybe some refactoring?